salt/modules/dockercompose.py

```diff
@@ -132,7 +133,7 @@ def __virtual__():
         match = re.match(VERSION_RE, str(compose.__version__))
         if match:
             version = tuple([int(x) for x in match.group(1).split('.')])
-            if MIN_DOCKERCOMPOSE >= version:
+            if version >= MIN_DOCKERCOMPOSE and version <= MAX_DOCKERCOMPOSE:
```

Comment:
Since you're at it, no logging should happen on `__virtual__()`, instead, return `(False, <logged reason>)`...